### DISCUSSION TITLE
programming language not detected in repository

### DISCUSSION THREAD

**Discussion by xenamakky** [github-linguist/linguist#5581] [Q&A]
Hey, I'm just starting to use GitHub more often, so sorry if this has been asked 47520348 times, I'm just not sure what to do when this happens. I usually see the language of others' repositories, but I don't always see them in mine.

**Comment by Nixinova** [comment]
Link to repo?

**Reply by xenamakky** [reply]
https://github.com/xenamakky/hungry-turtle-game
My bad, I see the programming language now. Maybe it just needed some time to detect. Thanks anyway!
